lib/config.js

```diff
@@ -47,6 +47,11 @@ const configBaseSchema = v.object({
       if (['app', 'test', 'vendor', 'assets'].indexOf(key) !== -1) {
         return 'was removed, use `config.paths.watched` instead';
       }
+    },
+    specifics: {
```

Comment:
yay, some flexibility of `skemata` came in very handy, allowing `files.statics` to be different from other `files.` settings